x/pki/keeper/msg_server_approve_revoke_x_509_root_cert.go

```diff
@@ -75,6 +75,9 @@ func (k msgServer) ApproveRevokeX509RootCert(goCtx context.Context, msg *types.M
 
 		// remove from subject -> subject key ID map
 		k.RemoveApprovedCertificateBySubject(ctx, msg.Subject, msg.SubjectKeyId)
+
+		// remove from subject key ID -> certificates map
+		k.RemoveApprovedCertificatesBySubjectKeyID(ctx, msg.SubjectKeyId)
```

Comment:
Please add a test to 'handler_test.go' to ensure that this method correctly modifies this state